presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -105,6 +105,15 @@ General Aggregate Functions
 
     Returns ``n`` smallest values of all input values of ``x``.
 
+.. function:: reduce_agg(inputValue T, initialState S, inputFunction(S, T, S), combineFunction(S, S, S)) -> S
```

Comment:
apparently this isn't how we write function signatures most of the times; i don't object, but i am not sure

Reply:
@findepi : I am following `reduce` signature: https://prestodb.io/docs/current/functions/array.html#reduce

I do note we seem to have different style when writing function signature for scalar and aggregate function docs.